src/index.js

```diff
@@ -16,13 +16,20 @@ import './images/banner1.png';
 import './images/banner2.png';
 import './images/text-banner.png';
 
-
+import domUpdates from './domUpdates.js';
 import data from './gamedata.js';
+import Game from './Game.js';
+import Player from './Player.js';
 
-console.log(data);
-
-console.log('This is the JavaScript entry file - your code begins here.');
 
+$(".submit-btn").on("click", startGame);
 
-//Create instances of player
-//Create instances of Game
\ No newline at end of file
+function startGame() {
+  const p1Name = $("#p1-name-input").val();
+  const p2Name = $("#p2-name-input").val();
+  const player1 = new Player(p1Name);
```

Comment:
As it's currently written, the Player class needs a second parameter of 1 or 2 that will become its playerNum property. I thought it would be nice to know which player was 1 or 2 inside the instance, but if don't think that property is necessary/will be used I'm cool with taking it out.

Reply:
I will add the number.